index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>HELLO MICROVERSE</h1>
```

Comment:
- [ ] [REQUIRED] Please, make sure to add a class to the `h1` tag as requested in project requirements. And then use that class to add the color to separate style from structure.

Reply:
okay! thanks for the review. Please should i edit in my code editor and commit/push again from same branch  or should i create a new one
how do i resend because i am done with the correction